### PR TITLE
Update README with intel about yoast_title_override

### PR DESCRIPTION
**Pull request by clemherreman**
@toutma I added the key configuration in the README, but could you add some more precision about this option, and what is its use ?

<3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,3 +240,10 @@ More informations about purify Twig filter: https://github.com/Exercise/HTMLPuri
 Enjoy.
 
 ![](https://media.giphy.com/media/nNxT5qXR02FOM/giphy.gif)
+
+##3.6 Wordpress Yoast SEO plugin
```
Comment (zalexki) [inline README.md]: ## and 3.6 need space between them

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ You can use `easywordpress:generate-files` to automatically:
  - create `web/boot.php` file
  - create `app/config/parameters.php` file
  - create `.env` file at root directory
- 
+
 
 For every page template you can use `easywordpress:generate-templates` to generate wp themes file using this annotation (do not generate twig file).
```
Comment (zalexki) [inline README.md]: Wrong command now this is `easywordpress:generate:templates`

**Comment by toutma** [comment]
Fixes on commands 
- Files generation
- Templates generation

File: README.md (hunk 1 above)
Comment (zalexki) [reply README.md]: @toutma you forgot about that :)

**Pull request merged**